### DISCUSSION TITLE
Parallel Shifting

### DISCUSSION THREAD

**Discussion by cgloeckner** [xournalpp/xournalpp#5938] [Q&A] [answered]
Hey folks,

is there a way to perform parallel shifting in terms of geometrical constructions using xournal++?

I'd like to perform the following steps:
1. Draw a line with a ruler :white_check_mark: 
2. Place one side of the triangle on that line :white_check_mark: 
3. Place another ruler or triangle at another side of the triangle and let it snap. :grey_question: 
4. Move the second ruler or triangle whilst snapped to get a line that's parallel to the original one. :grey_question: 

Greetings
glocke

**Comment by rolandlo** [comment, marked as answer]
No, it's not possible to use multiple geometry tools at once. But you can just select the line and move it (with the mouse, stylus or keyboard). The movement is always parallel.

**Reply by cgloeckner** [reply]
That's true. Unfortunately, I hoped for a way to demonstrate parallel shifting to my students :-)
